src/testclient/scripts/1.configureSpider/get-cloud.sh

```diff
@@ -48,6 +19,57 @@
     # for Cloud Driver Info
     curl -H "${AUTH}" -sX GET http://$SpiderServer/spider/driver/${DriverName[$INDEX]} | jq ''
     echo ""
+}
+
+#function get_cloud() {
+
+    echo "####################################################################"
+    echo "## 0. Get Cloud Connction Config"
+    echo "####################################################################"
+
+    source ../init.sh
+
+    echo "AUTH: $AUTH"
+    echo "TumblebugServer: $TumblebugServer"
+    echo "NSID: $NSID"
+    echo "INDEX: $INDEX"
+    echo "REGION: $REGION"
+    echo "{CONN_CONFIG[$INDEX,$REGION]}: ${CONN_CONFIG[$INDEX,$REGION]}"
+    echo "POSTFIX: $POSTFIX"
+    echo ""
+
+    if [ "${INDEX}" == "0" ]; then
+        echo "[Parallel excution for all CSP regions]"
+
+        INDEXX=${NumCSP}
+        for ((cspi = 1; cspi <= INDEXX; cspi++)); do
+            echo $i
+            INDEXY=${NumRegion[$cspi]}
+            CSP=${CSPType[$cspi]}
+            for ((cspj = 1; cspj <= INDEXY; cspj++)); do
+                # INDEX=$(($INDEX+1))
+
+                echo $j
+                INDEX=$cspi
+                REGION=$cspj
+                echo $CSP
+                echo $REGION
+                echo ${RegionName[$cspi,$cspj]}
+                
```

Comment:
ref) https://github.com/cloud-barista/cb-tumblebug/pull/714

```suggestion
    if [ "${INDEX}" == "0" ]; then
        echo "[Parallel excution for all CSP regions]"

        INDEXX=${NumCSP}
        for ((cspi = 1; cspi <= INDEXX; cspi++)); do
            INDEXY=${NumRegion[$cspi]}
            CSP=${CSPType[$cspi]}
            echo "[$cspi] $CSP details"
            for ((cspj = 1; cspj <= INDEXY; cspj++)); do
                echo "[$cspi,$cspj] ${RegionName[$cspi,$cspj]}" 
                
```

이런 식으로 출력을 간소화하면 어떨까요?